### ISSUE TITLE
Can not run init command with --metadatajson-urls option

### ISSUE THREAD

**Issue by naogify** [unvt/charites#32]
When I run the command,  

```
charites init style.yml --metadatajson-urls https://tileserver.example.com/metadata.json
```
then it shows error message.

```
error: missing required argument '--tilejson-urls'
```

**Comment by JinIgarashi** [comment]
@naogify fixed with #34 now. Could you confirm it?

**Comment by naogify** [comment]
@JinIgarashi Thank you! I approved it.
